app/assets/javascripts/visualizations/highvis/bar.coffee

```diff
@@ -142,8 +142,10 @@ $ ->
 
       drawControls: ->
         super()
-
-        @drawGroupControls(data.textFields)
+        # Remove group by number fields, only for pie chart
```

Comment:
What's the rational behind adding the feature to base vis and then removing it from all other vises as opposed to just adding it to pie.coffee?